core/os/os.h

```diff
@@ -179,6 +179,7 @@ class OS {
 	virtual Error shell_open(String p_uri);
 	virtual Error shell_show_in_file_manager(String p_path, bool p_open_folder = true);
 	virtual Error set_cwd(const String &p_cwd);
+	virtual String get_cwd() const;
```

Comment:
Note: neither `set_cwd`, not newly added `get_cwd` are bound, and therefore it will be accessible only from the engine C++ code (not from GDScript or GDExtension). If you want't it to be accessible, a wrapper should be added to the `CoreBind` (`OS` is a special case and its methods can't be bound directly).

Here's links to one of such wrappers, for example (`OS.get_name`):

Wrapper definition:
https://github.com/godotengine/godot/blob/9adb7c7d130c6d4eb0e80b92d6eebd71fac3384d/core/core_bind.h#L175

Wrapper implementation:
https://github.com/godotengine/godot/blob/9adb7c7d130c6d4eb0e80b92d6eebd71fac3384d/core/core_bind.cpp#L351-L353

Wrapper binding:
https://github.com/godotengine/godot/blob/9adb7c7d130c6d4eb0e80b92d6eebd71fac3384d/core/core_bind.cpp#L601

Reply:
I noticed that set_cwd is not bound, so I don't plan to get_cwd the corresponding one.

Reply:
I hope it will provide an intuitive way to get the current working directory in future engine development, and it will correspond to set_cwd.
Maybe it could also be used to gradually replace the original code that used DirAccess to get the working path?